xbmc/cores/VideoRenderers/BaseRenderer.cpp

```diff
@@ -222,11 +222,14 @@ void CBaseRenderer::FindResolutionFromFpsMatch(float fps, float& weight)
 RESOLUTION CBaseRenderer::FindClosestResolution(float fps, float multiplier, RESOLUTION current, float& weight)
 {
   RESOLUTION_INFO curr = g_graphicsContext.GetResInfo(current);
-
+  RESOLUTION_INFO orig = curr;
```

Comment:
the signature of the method is misleading. what is "current"? is that current resolution of gfx context or some current index of an array it iterates?